tests/functional/codegen/modules/test_storage_variables.py

```diff
@@ -124,3 +124,50 @@ def get_lib_counter() -> uint256:
 
     assert c.counter() == 1
     assert c.get_lib_counter() == 5
+
+
+def test_import_complex_types(get_contract, make_input_bundle):
+    lib1 = """
+an_array: uint256[3]
+a_hashmap: HashMap[address, HashMap[uint256, uint256]]
+
+@internal
+def set_array_value(ix: uint256, new_value: uint256):
+    self.an_array[ix] = new_value
+
+@internal
+def set_hashmap_value(ix0: address, ix1: uint256, new_value: uint256):
+    self.a_hashmap[ix0][ix1] = new_value
+    """
+
+    contract = """
+import lib
+
+initializes: lib
```

Comment:
Random thought: would `modname.initializes` be an accessible variable to a downstream user?

Reply:
no, you would get something like this
```
vyper.exceptions.UnknownAttribute: module_uses.vy has no member 'initializes'. 
```

Reply:
Not saying today, but would it make sense to be able to metaprogram by accessing `initializes` as the set of modules that a module is using somehow :thinking: